raiden_contracts/data/deployment_goerli.json

```diff
@@ -3,19 +3,19 @@
   "chain_id": 5,
```

Comment:
I'm not sure about the version above. Do we need to change this? It's not set in the 0.37 and 0.38 data dirs as well.

Reply:
It works with `get_contracts_deployment_info(5, "0.39.0")`

Reply:
I remember asking Karl about it before when making a release and he said it's just fine to leave it as `null`, nothing should really depend on it.

Reply:
Cool. good to know!